lib/posix-process/process.c

```diff
@@ -110,6 +110,8 @@ static inline pid_t find_free_tid(void)
 	}
 	if (found == TIDMAP_SIZE) {
 		/* no free PID */
+		uk_pr_err("Could not allocate TID: Out of TIDs (configured max: %d)\n",
```

Comment:
Hi @RaduNichita, yes, I added that message here while preserving the existing logic, but I think you're right. `git blame` shows that this pattern originates since inception of `process.c`, and until a1c73be910 both conditionals checked against `CONFIG_LIBPOSIX_PROCESS_MAX_PID`. Before making any changes I would like to check on the reasoning with @skuenzer who is the original author, but given that he's away this week I would prefer if we considered that change on a follow-up PR.

Reply:
Agree.